src/lib.rs

```diff
@@ -450,4 +452,62 @@ pub mod formats {
             }
         }
     }
+
+    #[cfg(feature = "cbor")]
+    mod cbor {
+        use super::*;
+
+        /// CBOR codec using [serde_cbor](https://docs.rs/serde_cbor) crate.
+        #[cfg_attr(feature = "docs", doc(cfg(cbor)))]
+        #[derive(Default)]
```

Comment:
Use derivative's `Default` impl.

Reply:
Out of curiosity what is the difference between derivative's `Default` and std lib's `Default`?